src/components/autonomous/WaypointNav.js

```diff
@@ -18,6 +20,17 @@ function WaypointNav() {
     dispatch(requestWaypointNav(formJson));
   };
 
+  function grabFromClipboard () {
+    navigator.clipboard.readText().then(text => {
+      if(text.match("-?\\d{1,}\\.?\\d{0,}, -?\\d{1,}\\.?\\d{0,}")) {
+        setLat(text.split(", ")[0]);
+        setLon(text.split(", ")[1]);
```

Comment:
Very minor, but no point in splitting the string twice. This could be done more cleanly:

```
const [lat, lon] = text.split(", ", 2);
setLat(lat);
setLon(lon);
```